stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -91,8 +61,6 @@ class HomeViewModel @Inject constructor(
         get() = _destinations
 
     // Toolbar section variables
-    private val _toolbarTitle = MutableStateFlow(TransactionType.DISTRIBUTION)
-    val toolbarTitle: StateFlow<TransactionType> get() = _toolbarTitle
 
     private val _fromFacility = MutableStateFlow(UIText.StringRes(R.string.from_facility))
```

Comment:
Why don't we move this to settingsuistate or create a toolbaruistate to listen this changes?